recipe/build.sh

```diff
@@ -16,6 +16,7 @@ autoreconf -fi
 ./configure --prefix="${PREFIX}" \
   ${CONFIGURE_ARGS} \
   --with-hdf5="${PREFIX}" \
+  --with-metis="${PREFIX}" \
```

Comment:
Shouldn't we only use METIS if it's an MPI build?

Reply:
Metis is fully serial. And `mbpart` can interface to it cleanly as an offline tool. This means you have to run `mbpart` serially, but at least it provides a way to pre-partition grids to run in parallel with other tools. Without this dependency, there is no way for a user to pre-partition with MOAB out-of-the-box.

Perhaps you are thinking of ParMetis ? @gonuke